src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what is this?

Reply:
Ah sorry forgot to mention in the PR description. It was needed for spying on final classes. But anyways, after your changes in the tests for builders its not needed anymore: 6eece0c